_posts/_posts/0000-01-02-fofolina.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
## change this lineee

![image](https://user-images.githubusercontent.com/86021537/122283019-63fc1280-ceec-11eb-8545-320147db4023.png)